test/ddlog

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# A helper script to play with DDlog command-line interface
+set -eu
+
+# find itself
+Self=$(readlink -f "$0" 2>/dev/null || {
+    # XXX readlink -f is only available in GNU coreutils
+    cd $(dirname -- "$0")
+    n=$(basename -- "$0")
+    if [ -L "$n" ]; then
+        L=$(readlink "$n")
+        if [ x"$L" != x"${L#/}" ]; then
+            echo "$L"; exit
+        else
+            cd "$(dirname -- "$L")"
+            n=$(basename -- "$L")
+        fi
+    fi
+    echo "$(pwd -P)/$n"
+})
+Here=$(dirname "$Self")
+
+# set up environment
+classpathFile="$Here"/../target/scala-2.10/classpath
```

Comment:
It'd be nice to unify the definition somehow, but that path is decided by SBT and the rest of our scripts are just conforming to it.